src/libsync/configfile.h

```diff
@@ -60,6 +61,7 @@ class OWNCLOUDSYNC_EXPORT ConfigFile
     [[nodiscard]] bool showConfigBackupWarning() const;
 
     bool exists();
+    bool isVisible();
```

Comment:
This is not descriptive enough -- what is visible? What isn't visible?

Reply:
As mentioned in PR description, use of this is- Some of components are not required in UI file for the magentacloud requirements, so instead of remove it from UI, it will show/hide from cpp file using configFile. (we have already discussed with @tobiasKaminsky  & @camilasan  for this).

So name is generic(will update in next commit with "isComponentVisible()") for entire code, same function will be use for other screen also wherever component need to be hidden.